LICENSE

```diff
@@ -564,4 +564,64 @@ ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
 CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 SOFTWARE.
 ========================================================================
+The libarchive distribution as a whole is Copyright by Tim Kientzle and others
```

Comment:
This should be converted to a "This project includes ... which is available under ..." phrasing in lieu of taking the associated words from the contributing license.